service/Dockerfile

```diff
@@ -6,21 +6,22 @@ FROM evaka-base:latest
 
 USER root
 RUN set -eux \
-    && apt-get update && apt-get -y --no-install-recommends install \
+ && apt-get update && apt-get -y --no-install-recommends install \
        curl \
        openjdk-11-jre-headless \
-    && rm -rf /var/lib/apt/lists/*
+ && rm -rf /var/lib/apt/lists/*
 
 USER evaka
 WORKDIR /home/evaka
 
 COPY --chown=evaka:evaka "entrypoint.sh" "entrypoint.sh"
-RUN chmod +x "entrypoint.sh"
 COPY --chown=evaka:evaka "target/org/" "org/"
 COPY --chown=evaka:evaka "target/META-INF" "META-INF/"
 COPY --chown=evaka:evaka "target/BOOT-INF/lib" "BOOT-INF/lib/"
 COPY --chown=evaka:evaka "target/BOOT-INF/classes" "BOOT-INF/classes/"
 
+RUN curl -sSfo dd-java-agent.jar "https://repo1.maven.org/maven2/com/datadoghq/dd-java-agent/0.9.0/dd-java-agent-0.9.0.jar"
+
```

Comment:
`0.9.0` seems to be a 2018 version while latest is `0.89.0`

Reply:
Good point. I'll change that.